src/main/java/org/hisp/dhis/random/EnrollmentRandomizer.java

```diff
@@ -52,27 +52,30 @@
 
     private int minEVent;
 
-    private EventRandomizer eventRandomizer = new EventRandomizer();
+    private final EventRandomizer eventRandomizer;
 
-    public EnrollmentRandomizer( int minEVent, int maxEvent )
+    private final Randomizer rnd;
+
+    public EnrollmentRandomizer( int minEVent, int maxEvent, Randomizer rnd )
```

Comment:
I would move parameter `Randomizer rnd` first as `int minEVent, int maxEvent` are kind of optional due to the `EnrollmentRandomizer(Randomizer rnd)` constructor. This way the order across constructors is preserved 😄. This might also make sense for other classes for consistency.